apps/portal/controllers/error-pages/error405.html

```diff
@@ -17,7 +33,20 @@
 <i class="icon-unlink error-icon"></i>
 <h1>Error 405</h1>
 <h3>Method not allowed.</h3>
-<h4><a href="/portal">Go to Login Page</a></h4>
+<div id="portal" style="visibility: visible; display: block; ..">
+    <h4><a href="/portal">Go to Portal</a></h4>
+</div>
+<script type="text/javascript">
+    var link = document.getElementById('portal');
```

Comment:
Thank you for pointing out. Error pages were refactored and these code segments were removed already.